etna/lib/etna/application.rb

```diff
@@ -13,9 +13,16 @@ def self.included(other)
   end
 
   def self.find(klass)
-    Kernel.const_get(
-      klass.name.split('::').first
-    ).instance
+    namespace = klass.name.split('::').first
+    if (namespace_klass = Kernel.const_get(namespace)) && (namespace_klass.respond_to? :instance)
+      return namespace_klass.instance
+    end
+
+    if (app_name = ENV['APP_NAME']) && (app_klass = Kernel.const_get(app_name.capitalize)) && (app_klass.respond_to? :instance)
```

Comment:
These changes make it possible to use Etna::Command subclasses that are not specific to any given app namespace by looking up the app class via `APP_NAME` environment variable.